.github/workflows/push.yml

```
@@ -164,15 +169,17 @@ jobs:
 
     outputs:
-      CACHE_KEYS: ${{ steps.get-cache.outputs.keys }}
       TEST_CHUNKS: ${{ steps.test_projects.outputs.CHUNKS }}
       DOCKER_TAG: ${{ steps.docker_tags.outputs.DOCKER_TAG }}
       NODE_IMAGE_VERSION: ${{ steps.nodejs_image.outputs.NODE_IMAGE_VERSION }}
       LAST_GOOD_BUILD_DOCKER_TAG: ${{ steps.git_nx_base.outputs.LAST_GOOD_BUILD_DOCKER_TAG }}
       UNAFFECTED: ${{ steps.unaffected.outputs.UNAFFECTED }}
       BUILD_CHUNKS: ${{ steps.build_map.outputs.BUILD_CHUNKS }}
+      DOCKER_MATRIX: ${{ steps.docker_matrix.outputs.DOCKER_MATRIX }}
       IMAGES: ${{ steps.deploy_map.outputs.IMAGES }}
       node-modules-hash: ${{ steps.calculate_node_modules_hash.outputs.node-modules-hash }}
```

actionlint (1.7.4)

property "calculate_node_modules_hash" is not defined in object type {build_map: {conclusion: string; outcome: string; outputs: {string => string}}; deploy_map: {conclusion: string; outcome: string; outputs: {string => string}}; docker-deps: {conclusion: string; outcome: string; outputs: {string => string}}; docker_matrix: {conclusion: string; outcome: string; outputs: {string => string}}; docker_tags: {conclusion: string; outcome: string; outputs: {string => string}}; git_nx_base: {conclusion: string; outcome: string; outputs: {string => string}}; git_nx_head: {conclusion: string; outcome: string; outputs: {string => string}}; git_refs: {conclusion: string; outcome: string; outputs: {string => string}}; nodejs_image: {conclusion: string; outcome: string; outputs: {string => string}}; restore-cache: {conclusion: string; outcome: string; outputs: {string => string}}; slack: {conclusion: string; outcome: string; outputs: {}}; test_projects: {conclusion: string; outcome: string; outputs: {string => string}}; unaffected: {conclusion: string; outcome: string; outputs: {string => string}}; yarn-cache-dir-path: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)